projects/aca-shared/src/lib/components/page-layout/page-layout.component.html

```diff
@@ -1,3 +1,11 @@
-<ng-content select="aca-page-layout-header"></ng-content>
-<ng-content select="aca-page-layout-error" *ngIf="hasError"></ng-content>
-<ng-content select="aca-page-layout-content" *ngIf="!hasError"></ng-content>
+<div class="aca-content-container" [ngClass]="{'aca-page-layout-container' : hideSidenav}">
+    <div class="aca-content-header">
+        <div class="aca-menu-icon" [ngClass]="{'aca-display-menu' : !hideSidenav}" (click)="toggleClick()" (keypress)="toggleClick()">
```

Comment:
your class "aca-display-menu" actually hides content, so you say hide content when hideSidenav = false ? this is just stupid